lib/generators/sufia/templates/config/sufia.rb

```diff
@@ -120,7 +120,14 @@
   # NOTE: if you have always sent analytics to GA for downloads and page views leave this commented out
   # config.analytic_start_date = DateTime.new(2014,9,10)
 
-  config.geonames_username = ''
+  # Location autocomplete uses geonames to search for named regions.
+  # Specify the user for connecting to geonames:
+  # config.geonames_username = ''
+
+  # Should the acceptance of the licence agreement be active (checkbox), or
```

Comment:
Folks who are upgrading should at least know this is now possible with 7.0.0, though, if they want to turn off active acceptance. We should at least include this in the release notes (or ticket doing same), and then if someone wants to add this to the upgrade generator, great, but not necessary.